src/materialized/src/bin/materialized/main.rs

```diff
@@ -151,9 +151,9 @@ struct Args {
     #[structopt(long, hide = true)]
     persist_storage_enabled: bool,
 
-    /// Enable persistent Kafka UPSERT source. Has to be used with --experimental.
+    /// Enable persistent Kafka source. Has to be used with --experimental.
     #[structopt(long, hide = true)]
-    persistent_kafka_upsert_source: bool,
+    persistent_kafka_source: bool,
```

Comment:
ugh, I'm so sorry to ask this after you've already typed up this PR, but could we make this `persistent_kafka_sources` (plural "sources") to match `persistent_user_tables`?

Reply:
Of course, no worries!